test/fixtures/project/hokusai/config.yml

```diff
@@ -3,3 +3,5 @@ project-name: hello
 pre-deploy: migrate.sh
 post-deploy: sh -c report.sh
 git-remote: git@github.com:artsy/hokusai.git
+template-config-files:
```

Comment:
I wonder what the use case for this file is-- is it necessary for local dev, is to demonstrate that we can have a local set of vars?